minigalaxy/installer.py

```diff
@@ -167,6 +170,35 @@ def copy_thumbnail(game):
     return error_message
 
 
+def create_applications_file(game):
+    error_message = ""
+    preferences_switch = Config.get("create_applications_file")
+    if game.platform == "linux" and preferences_switch:
+        path_to_shortcut = os.path.join(APPLICATIONS_DIR, game.name+".desktop")
```

Comment:
This is really cosmetic, but can you please change it to:
"
path_to_shortcut = os.path.join(APPLICATIONS_DIR, "{}.desktop".format(game.name))
"

Reply:
Done in all places within the file.